infra/examples-dev/aws-all/google-workspace-variables.tf

```diff
@@ -1,6 +1,7 @@
 variable "google_workspace_gcp_project_id" {
   type        = string
   description = "string ID of GCP project that will host oauth clients for Google Workspace API connectors; must exist"
+  default = null
```

Comment:
I don't think having the default is great here, bc leaving it empty isn't the proper solution. for real use, you should just delete this file from your configuration if not using Google Workspace sources - that's why split into separate file.